docs/api.md

```diff
@@ -1,468 +1,427 @@
 # CF K8s API Endpoint Documentation
 
-**This document captures API endpoints currently supported by the shim**
+This document lists all the CF API endpoints supported by Korifi and their parameters.
 
-## Resources
+## Cloud Foundry V3 API
 
-### Root
+### [Apps](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#apps)
 
-Docs: https://v3-apidocs.cloudfoundry.org/version/3.117.0/index.html#root
+#### [Create an app](https://v3-apidocs.cloudfoundry.org/version/3.117.0/#create-an-app)
```

Comment:
Nothing we can do about it, but a big gripe I have with the V3 API docs is that there is no way to link to the latest release.

Reply:
Yeah 😞 I'll still update all links to point to the current latest!

Reply:
Wow that's great!